tests/gemini_test.yaml

```diff
@@ -13,9 +13,9 @@ store_results_in_elasticsearch: False
 # gemini
 # cmd: gemini -d -n [NUM_OF_TEST_ITERATIONS] -c [NUM_OF_THREADS] -p [NUM_OF_PARTITION_KEYS_PER_THREAD] -m mixed -f
 # the below cmd runs about 3 hours
-gemini_cmd: "gemini -d -n 500000 -c 20 -p 100 -m mixed -f"
-gemini_url: 'https://s3.amazonaws.com/scylla-gemini/Binaries/gemini-latest'
-gemini_static_url: 'https://s3.amazonaws.com/scylla-gemini/Binaries/schema.json'
+gemini_cmd: "gemini -d --duration 120s -c 20 -p 100 -m mixed -f"
```

Comment:
why duration is different  from  `test-cases/gemini/gemini-basic-3h.yaml`?

Reply:
fixed